lib/dojos.js

```diff
@@ -406,6 +410,7 @@ exports.register = function (server, options, next) {
           expectedAttendees: Joi.number().allow(null),
           website: joiValidator.optionalUri(),
           twitter: Joi.alternatives().try(joiValidator.twitter(), Joi.string().valid(null).valid('')),
+          facebook: Joi.alternatives().try(joiValidator.facebook(), Joi.string().valid(null).valid('')),
```

Comment:
We've already allowed `''` in the facebook validator, so probably don't need `.valid('')` here.